shared/macros-oval.jinja

```diff
@@ -218,6 +218,32 @@
   </unix:file_test>
 {{%- endmacro %}}
 
+{{#
+  Macro to define the OVAL criterion that requires a file not to exist.
+  This macro can take one parameter:
+    - filepath (String): Path to the file to be checked.
+    - stem of the test name
+#}}
+{{%- macro oval_config_file_absent_criterion(filepath, stem) -%}}
+	<criterion comment="Pass if {{{ filepath }}} does not exist" test_ref="test_{{{ stem }}}" />
+{{%- endmacro %}}
+
+{{#
+  Macro to define the OVAL test to check if the configuration file exists (Test definition).
+  This macro can take one parameter:
+    - filepath (String): Path to the configuration file to be checked.
+    - test_version: OVAL test version metadata
+    - object_version: OVAL object version metadata
+#}}
+{{%- macro oval_config_file_absent(filepath, stem, test_version=1, object_version=1) -%}}
```

Comment:
I suggest ignoring the `version` attributes and always set them to 1. Then, the macro could have only 2 parameters.

Reply:
OK.